Gordon360/Services/HousingService.cs

```diff
@@ -159,10 +159,10 @@ public int SaveApplication(string userID, string sess_cde, string editorID, Apar
             DateTime now = System.DateTime.Now;
 
             SqlParameter sessionParam = new SqlParameter("@SESS_CDE", sess_cde);
-            SqlParameter editorParam = new SqlParameter("@STUDENT_ID", editorID);
+            SqlParameter editorParam = new SqlParameter("@STUDENT_USERNAME", editorUsername);
 
             // If an application ID was not passed in, then check if an application already exists
-            idResult = RawSqlQuery<ApartmentAppIDViewModel>.query("GET_AA_APPID_BY_STU_ID_AND_SESS @SESS_CDE, @STUDENT_ID", sessionParam, editorParam); //run stored procedure
+            idResult = RawSqlQuery<ApartmentAppIDViewModel>.query("GET_AA_APPID_BY_STU_ID_AND_SESS @SESS_CDE, @STUDENT_USERNAME", sessionParam, editorParam); //run stored procedure
```

Comment:
Same as above, this name is outdated. In the future, you could copy the SP with the new name, and then rename it in the API, and once the API is updated everywhere, delete the old SP.

Reply:
Oh thank you for that advice, that will make it much easier to change things over